### DISCUSSION TITLE
How to use `:Tutor` in NvChad?

### DISCUSSION THREAD

**Discussion by lamualfa** [NvChad/NvChad#2055] [Q&A] [answered]
I am trying to run the `:Tutor` command but it tells me that the command does not exists. How to get back the `:Tutor` command without removing the NvChad?

**Comment by siduck** [comment, marked as answer]
@lamualfa as-salamu alaikum, check nvim --clean for now or enable Tutor in lazy_nvim config ( check core/default_configs.lua)

**Reply by lamualfa** [reply]
> @lamualfa as-salamu alaikum, check nvim --clean for now or enable Tutor in lazy_nvim config ( check core/default_configs.lua)

Waalaikumsalam. Thank you for the response.

> or enable Tutor in lazy_nvim config ( check core/default_configs.lua)

How to enable it? I can't find any `Tutor` configuration in there.

Edit

Ok, I found it.

https://github.com/NvChad/NvChad/blob/582dff9458ee8ac70149583829b6f8faecdc3aef/lua/plugins/configs/lazy_nvim.lua#L36
